package.json

```diff
@@ -93,12 +101,12 @@
     "output-file-sync": "^1.1.1",
     "phantomjs-prebuilt": "^2.1.7",
     "portfinder": "^1.0.3",
-    "react": "^15.0.0",
-    "react-component-metadata": "^2.1.1",
-    "react-dom": "^15.0.0",
+    "react": "^15.0.1",
+    "react-component-metadata": "^3.0.0",
```

Comment:
@jquense Here's the error log:
```
Error: Command failed: npm run docs-build -- 
TypeError: Converting circular structure to JSON
    at Object.stringify (native)
    at Object.stringify (/Users/bjorn/dev/react-bootstrap/node_modules/core-js/modules/es6.symbol.js:138:21)
    at stringify (/Users/bjorn/dev/react-bootstrap/node_modules/core-js/library/fn/json/stringify.js:4:26)
    at [object Object].render (Root.js:36:30)
    at [object Object].ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext (/Users/bjorn/dev/react-bootstrap/node_modules/react/lib/ReactCompositeComponent.js:679:34)
    at [object Object].ReactCompositeComponentMixin._renderValidatedComponent (/Users/bjorn/dev/react-bootstrap/node_modules/react/lib/ReactCompositeComponent.js:699:32)
    at [object Object].wrapper [as _renderValidatedComponent] (/Users/bjorn/dev/react-bootstrap/node_modules/react/lib/ReactPerf.js:66:21)
    at [object Object].ReactCompositeComponentMixin.performInitialMount (/Users/bjorn/dev/react-bootstrap/node_modules/react/lib/ReactCompositeComponent.js:284:30)
    at [object Object].ReactCompositeComponentMixin.mountComponent (/Users/bjorn/dev/react-bootstrap/node_modules/react/lib/ReactCompositeComponent.js:237:21)
    at [object Object].wrapper [as mountComponent] (/Users/bjorn/dev/react-bootstrap/node_modules/react/lib/ReactPerf.js:66:21)
```

Looks like it gets the component metadata all wrong...